src/run/run.ts

```diff
@@ -2,10 +2,36 @@ import { getBooleanInput, getInput, setFailed, setOutput } from "@actions/core";
 import { INewTestRun } from "testrail-api";
 import { extractError } from "../utils";
 import { Environment, InputKey, RunInputs, TestRunConfig, TestRailOptions } from "./run.definition";
-import { extractTestResults, getTrunkTestRunConfigs } from "./run.utils";
+import { containsE2Etest, extractTestResults, getTrunkTestRunConfigs, getUnitTestConfig } from "./run.utils";
 import { TestrailService } from "../services";
 import findDuplicates from "../utils/findDuplicate";
 
+async function closeMilestoneWithOnlyUnitTest(
```

Comment:
i would move this function below `run`, as `run` is the primary function here and should be read first. Perhaps right above or below `reportToTestrail`.

Reply:
Done in https://github.com/q4mobile/stratus-testrail-reporter/pull/33/commits/0ef07600b532e8ef013028d78fe9c0b4e574f0cb.